src/einsteinpy/symbolic/predefined/bessel_gravitational_wave.py

```diff
@@ -0,0 +1,44 @@
+from einsteinpy.symbolic import constants
+from einsteinpy.symbolic.metric import MetricTensor
+from sympy import Function, Symbol, besselj, cos, diag, exp, sin, sqrt, symbols
+
+
+def Bessel_Gravitational_Wave(C=symbols("C")):
```

Comment:
Same:)

And please take care of all the instances below. Sorry for pointing out so teeny-tiny discrepancies but they are to maintain consistency.

Reply:
No worries! I think it is important too, I just missed it haha